mapstory/templates/_header.html

```diff
@@ -1,6 +1,9 @@
 {% load i18n avatar_tags %}
 <header class="navigation" role="banner">
     <div class="navigation-wrapper container">
+        {% if THEME == 'blue' %}
+            <img src="{{ STATIC_URL }}mapstory/img/storyscapes_banner_logo.png" height="50px"/>
```

Comment:
What happens here if the django admin also adds a logo?

Reply:
If we're not using the django admin logo, we'll need to replace all of the places that use it.
![image](https://user-images.githubusercontent.com/1152120/39540542-c78d3488-4df7-11e8-8d74-ba93c16283af.png)